test/e2e/lib/framework/serverApps/mock.ts

```diff
@@ -47,24 +47,28 @@ export function createMockServerApp(endpoints: Endpoints, setup: string) {
   })
 
   app.get('/', (_req, res) => {
+    res.header(
+      'Content-Security-Policy',
+      `connect-src ${intakeUrl}; script-src 'self' 'unsafe-inline'; worker-src blob:;`
```

Comment:
* There is only one intake host, so no need for a star in connect-src
* We host the SDK bundles on the same domain as the app, so we can use `'self'` for `script-src`